### PR TITLE
Fixed pretty formatting of first cfbs init commit

### PR DESCRIPTION
**Pull request by olehermanse**
- De-duplicated cfbs sorting rules
- Renamed sorting rule variables
- Fixed pretty formatting of first cfbs init commit
- Updated imports for top level keys and module keys

### CODE REVIEW

**Review by craigcomstock — APPROVED** [review]

File: cfbs/commands.py [hunk 1]
```diff
@@ -95,34 +100,7 @@ def pretty_command(filenames: list, check: bool, keep_order: bool) -> int:
     if not filenames:
         user_error("Filenames missing for cfbs pretty command")
 
-    cfbs_sorting_rules = None
```
Comment (oleorhagen) [inline cfbs/commands.py:98]: thx. forgot this one

File: cfbs/commands.py [hunk 2]
```diff
@@ -207,7 +209,10 @@ def init_command(index=None, masterfiles=None, non_interactive=False) -> int:
 
     config["git"] = do_git
 
-    write_json(cfbs_filename(), config)
+    data = pretty(config, CFBS_DEFAULT_SORTING_RULES) + "\n"
```
Comment (oleorhagen) [inline cfbs/commands.py:213]: Q: Why not write it through the `cfbs_json` object like we do most other things?

Comment (olehermanse) [reply cfbs/commands.py:213]: Feel free to take a stab at it, shouldn't be **very** hard.

File: cfbs/commands.py (hunk 1 above)
Comment (olehermanse) [reply cfbs/commands.py:98]: np

**Pull request merged**